### ISSUE TITLE
 No available drivers to communicate with devices

### ISSUE THREAD

**Issue by lnanhkhoa** [mapaiva/react-native-usbserial#8]
`I/ReactNativeJS( 2851): [ { deviceName: '/dev/bus/usb/001/003',`
`I/ReactNativeJS( 2851):     vendorId: 911,`
`I/ReactNativeJS( 2851):     productId: 1345,`
`I/ReactNativeJS( 2851):     deviceId: 1003,`
`I/ReactNativeJS( 2851):     name: '/dev/bus/usb/001/003' },`
`I/ReactNativeJS( 2851):   { deviceName: '/dev/bus/usb/004/003',`
`I/ReactNativeJS( 2851):     vendorId: 11388,`
`I/ReactNativeJS( 2851):     productId: 293,`
`I/ReactNativeJS( 2851):     deviceId: 4003,`
`I/ReactNativeJS( 2851):     name: '/dev/bus/usb/004/003' } ]`
 No available drivers to communicate with devices

Why i can not connect devices? how should i do?

**Comment by ovidius72** [comment]
I'm having the same issue. Any suggestion ?

**Comment by omatrot** [comment]
Yes, take a look at this [fork/commit](https://github.com/EarthlingInteractive/react-native-usbserial/commit/d60463a973153445fd1ace6bba5ddc423d9486ca)
Please note that the driver is hardcoded to FtdiSerialDriver line 86 of ReactUsbSerialModule.java 
If this driver is not working for you, you can use another one to see if that suits you..
In m case, I'm using CdcAcmSerialDriver instead.
